Shashwat-Gandhi/README.md

```diff
@@ -5,7 +5,8 @@ CSE 1st Year MNIT Jaipur <br>
 - [ ] Reach a score of 2200 on hackerrank in problem solving  <br>
 - [ ] Make around 5 to 10 games using Unity
 - [x] Game 1 : "2048" game (with unity) <br>
-- [x] Game 2 : colorFill3D game (with unity) <br><br>
+- [x] Game 2 : colorFill3D game (with unity) <br>
+- [ ] Game 3 : Smash Hit <br<br>
 
```

Comment:
Instead of using `<br>`, you can go to next line by giving a gap on another line